app/views/members/autocomplete_for_member.json.erb

```diff
@@ -27,16 +27,31 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
+<%#
+  Lists found users (principals) to be used for the select box.
+  If the user enters an email address, a 'result' in the form of
+  'Invite: <email>' will be returned.
+#%>
+
 {
   "results":
   {
+    <%# Offer to invite via email unless it's one of the found user's addresses already. #%>
+    <% email = params[:q] =~ /.+@.+/ && @principals.none? { |p| p.mail == params[:q] } %>
```

Comment:
Then perfectly valid [email addresses](https://en.wikipedia.org/wiki/Email_address#Valid_email_addresses) will be not supported, though! 

> admin@mailserver1 (local domain name with no TLD)

But yes, I suppose that's exceedingly rare.